<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPay</class>
 <widget class="QWidget" name="UserPay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;会员缴费&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResearchBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入身份证：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RechangeBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>充值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MoneyLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MoneyPayLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入金额：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上层</string>
   </property>
  </widget>
  <widget class="QLabel" name="BRGLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/backimage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="UserInfoTextEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>300</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <zorder>BRGLabel</zorder>
  <zorder>TitleLabel</zorder>
  <zorder>ResearchBtn</zorder>
  <zorder>IDLineEdit</zorder>
  <zorder>IDLabel</zorder>
  <zorder>RechangeBtn</zorder>
  <zorder>MoneyLineEdit</zorder>
  <zorder>MoneyPayLabel</zorder>
  <zorder>returnBtn</zorder>
  <zorder>UserInfoTextEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
